### PR TITLE
Change GRIB2 parameter names and vertical levels for ocean/ice post

### PR DESCRIPTION
**Pull request by GwenChen-NOAA**
# Description

  Based on users' feedback, this PR do the following:
  1. Change GRIB2 parameter names DLWRF -> NLWRF and DSWRF -> NSWRF
  2. Change the vertical level of ocean 3D variables (WTMP, SALIN, UOGRD, and VOGRD) from "%g m below water surface" to "%g m below sea level"
  3. Round up depth numbers to integer (e.g. 4481.0625 -> 4481 m)

# Type of change
- Update

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?
Tested on WCOSS2 using [nc2grb](https://github.com/NOAA-EMC/gfs-utils/tree/develop/ush/nc2grb) scripts.

# Checklist
- [X] Any dependent changes have been merged and published
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: ush/oceanice_nc2grib2.sh
Comment (aerorahul) [inline ush/oceanice_nc2grib2.sh:48]: Just a note here that this (and the others below) are commented out.

**Review by aerorahul — COMMENTED** [review]
looks ok to me.
just one note on some of the variables that were `???` were commented out and still are.
Some other variables that were renamed e.g. `ALBICE` are still commented out.

File: ush/oceanice_nc2grib2.sh
Comment (GwenChen-NOAA) [reply ush/oceanice_nc2grib2.sh:48]: Yes, it is intentionally to put on hold. This is a todo item for future update.

**Comment by GwenChen-NOAA** [comment]
> looks ok to me. just one note on some of the variables that were `???` were commented out and still are. Some other variables that were renamed e.g. `ALBICE` are still commented out.

These variables are not available in wgrib2 v2.0.8. They will be added in another update.

**Review by aerorahul — APPROVED** [review]

**Review by WalterKolczynski-NOAA — APPROVED** [review]
Conditionally approved pending successful completion of CI test.

**Comment by WalterKolczynski-NOAA** [comment]
@GwenChen-NOAA Please merge in develop to get the CI fixes, then I think we are ready for CI testing.

**Comment by GwenChen-NOAA** [comment]
> @GwenChen-NOAA Please merge in develop to get the CI fixes, then I think we are ready for CI testing.

Just did.

**Comment by aerorahul** [comment]
Running the CI is no use here since we know its going to give all zeros.

**Pull request merged**
